packages/io/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "kuai io layer",
   "homepage": "https://github.com/ckb-js/kuai#readme",
   "license": "MIT",
-  "main": "lib/io.js",
```

Comment:
The `exports` field is recommended prior to the `main` fields

ref: https://nodejs.org/api/packages.html#package-entry-points